src/webgpu/shader/execution/robust_access_vertex.spec.ts

```diff
@@ -417,16 +498,109 @@ g.test('vertexAccess')
     pass.setPipeline(pipeline);
 
     // Run the draw variant
-    draw.insertInto(pass, p.indexed, p.indirect);
+    drawCall.insertInto(pass, isIndexed, isIndirect);
 
     pass.endPass();
-    t.device.queue.submit([encoder.finish()]);
+    this.device.queue.submit([encoder.finish()]);
 
     // Validate we see green instead of red, meaning no fragment ended up on-screen
-    t.expectSinglePixelIn2DTexture(
+    this.expectSinglePixelIn2DTexture(
       colorAttachment,
       'rgba8unorm',
       { x: 0, y: 0 },
       { exp: new Uint8Array([0x00, 0xff, 0x00, 0xff]), layout: { mipLevel: 0 } }
     );
+  }
+}
+
+export const g = makeTestGroup(F);
+
+g.test('vertex_buffer_access')
+  .params(u =>
+    u
+      .combineWithParams([
+        { indexed: false, indirect: true },
+        { indexed: true, indirect: false },
+        { indexed: true, indirect: true },
+      ])
+      .expand(
+        'drawCallTestParameter',
+        p =>
+          p.indirect
+            ? p.indexed
+              ? ([
+                  'indexCount',
+                  'instanceCount',
+                  'firstIndex',
+                  'baseVertex',
+                  'firstInstance',
+                  'vertexCountInIndexBuffer',
+                ] as const) // For drawIndexedIndirect
+              : (['vertexCount', 'instanceCount', 'firstVertex', 'firstInstance'] as const) // For drawIndirected
+            : (['baseVertex', 'vertexCountInIndexBuffer'] as const) // For drawIndexed
+      )
```

Comment:
Easier to understand IMO:
```suggestion
      .expand(
        'drawCallTestParameter',
        function* (p) {
          if (p.indexed) {
            yield* ['baseVertex', 'vertexCountInIndexBuffer'] as const;
            if (p.indirect) yield* ['indexCount', 'instanceCount', 'firstIndex', 'firstInstance'] as const;
          } else {
            if (p.indirect) yield* ['vertexCount', 'instanceCount', 'firstVertex', 'firstInstance'] as const;
          }
        })
```

Reply:
Done.